Makefile

```
@@ -30,17 +29,17 @@ check-deps:
 	endif
 		@echo "[make] LLVM is correctly set at $(MLIR_SYS_180_PREFIX)."
 
-deps:
+deps: check-deps install-nextest
```

We shouldn't check deps before installing them.
```suggestion
deps: install-nextest
```